lib/core/src/send_swap.rs

```diff
@@ -72,6 +72,7 @@ impl SendSwapStateHandler {
                             tx_id: lockup_tx_id.clone(),
                             timestamp: None,
                             amount_sat: swap.payer_amount_sat,
+                            fees_sat: 0, // Will be overwritten once LWK fetches the real tx
```

Comment:
Why not writing here the real fee, we create the tx so we should know it.

Reply:
Done: cabe96f48dc6d86d5a0aea9a32ae3b59c5297a0f

Although I don't think it brings much. This is a Send via swap, meaning the fees are anyway stored in the Swap table, so `list_payments` retrieves them from there. This field here is simply fallback for when we don't have the swap data and we need to show the tx fees, like for direct onchain payments. That case is covered here

https://github.com/breez/breez-liquid-sdk/pull/265/files#diff-f09a3f89b9a7fd7422e34d3a519a8ab878445ab33debc6bc60870fd348347a90R670-R679

and it already sets the tx fees.